docs/architecture.md

```diff
@@ -9,9 +9,14 @@ _Some incomplete notes_
 * **Driver** - Interfaces with [Chrome Debugging Protocol](https://developer.chrome.com/devtools/docs/debugger-protocol)  ([API viewer](https://chromedevtools.github.io/debugger-protocol-viewer/))
 * **Gatherers** - Uses Driver to collect information about the page. Minimal post-processing.
   * **Artifacts** - output of a gatherer
-* **Audits** - Using the Artifacts as input, Audits evaluate a test and assign pass/fail/scoring.
+* **Audit** - Tests for a single feature/optimization/metric. Using the Artifacts as input, an audit evaluates a test and  Resolves to a score, which may be pass/fail/numeric.  Meta info may contain markdown for rich text (code, links, etc).
   * **Computed Artifacts** - Generated on-demand from artifacts, these add additional meaning, and are often shared amongst multiple audits.
-* **Categories** - Grouping audit results into a user-facing section of the report (eg. `Best Practices`). Applies weighting and overall scoring to the section.
+  
+### Audit/Report terminology
+* **Audit Category** - Roll-up collection of audits and audit groups into a user-facing section of the report (eg. `Best Practices`). Applies weighting and overall scoring to the section. Examples: PWA, A11y, SEO.
```

Comment:
do we want to use at least one example that isn't an acronym 😆

Reply:
done